examples/step_operator_remote_training/setup.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -Eeo pipefail
+
+pre_run () {
+  zenml integration install aws sagemaker sklearn
+}
+
+pre_run_forced () {
+  zenml integration install aws s3 sagemaker sklearn -f
+}
```

Comment:
Why does one of these have s3 and the other one doesn't? Does it have something to do with out ci/cd actions?